src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -455,7 +455,8 @@ public void testPullPrivateRepoWithoutAuth() throws Exception {
   public void testBuildImageIdWithBuildargs() throws Exception {
     requireDockerApiVersionAtLeast("1.21", "build args");
 
-    final String dockerDirectory = Resources.getResource("dockerDirectoryWithBuildargs").getPath();
+    final String dockerDirectory = 
+        Paths.get(Resources.getResource("dockerDirectoryWithBuildargs").toURI()).toString();
```

Comment:
Yeah, i did that on purpose...  I find it easier to get my pull requests accepted (across many different projects) if i minimize my changes and adopt the approach of the project.  So i changed each line, but definitely agree that creating a helper method makes more sense.  Let me do so real quick.  Should have an update in a little while.